eland/series.py

```diff
@@ -468,6 +470,37 @@ def isin(self, other):
         else:
             raise NotImplementedError(other, type(other))
 
+    def isna(self):
```

Comment:
Let's also add the aliases `.isnull()` and `.notnull()` for `isna()` and `.notna()`

Reply:
Done